recipes/conjure-client/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Given that the tarball ships with a license we don't need this here

Reply:
actually nm, you probably still need this.  You may want to raise a pull request upstream to get the license added to the `MANIFEST.in`